Lesson_2/task_6.js

```diff
@@ -0,0 +1,7 @@
+// Реализовать функцию для перемешивания элементов массива
+
+function arrayShuffle(array) {
+    return array.sort(() => Math.round(Math.random()) * 2 - 1)
```

Comment:
Можно упростить вычисления:
```suggestion
    return array.sort(() => Math.round(Math.random()) - 0.5)
```